README.md

```diff
@@ -12,123 +12,9 @@ Under AB-375, California residents have the right to:
 The following is an alphabetical ordered list of companies and URLs to perform a Data Subject Access Request (DSAR) and exercise your rights under this law:
 
 # Directory
```

Comment:
Maybe make this a link or similar to the rendered version? [Directory](https://caprivacy.me)